diff --git a/helm/azure-workload-identity-webhook/templates/_helpers.tpl b/helm/azure-workload-identity-webhook/templates/_helpers.tpl
index 90017db..928b35d 100644
--- a/helm/azure-workload-identity-webhook/templates/_helpers.tpl
+++ b/helm/azure-workload-identity-webhook/templates/_helpers.tpl
@@ -40,6 +40,7 @@ helm.sh/chart: {{ include "workload-identity-webhook.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
 {{- end }}
 
 {{/*
